scripts/Make_Callrate_Report.py

```diff
@@ -1,3 +1,5 @@
+#!/bin/python
```

Comment:
```suggestion
#!/usr/bin/env python
```

Reply:
It is relatively safe to use a hardcoded path to the Bash interpreter as we have that on all machines. Python is a different story tough and there are major differences between Python 2 and 3. If you use
`module load Python/some-specific-version`
and then a script that starts with
`#!/usr/bin/env python`
, the script will run with the Python version loaded by the module system and fetched from the environment.
If you instead use the hard coded path /bin/python, then the script will run with the Python version installed by the Linux distro and ignore whatever was `module load`ed.